### DISCUSSION TITLE
guide-to-cooking/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#140] [Giscus]
# guide-to-cooking/

这篇文章介绍了一个万能炒菜公式。作者指出，做菜没有固定的用量，而是根据个人口味和情况来调整。对于炒任何菜，可以先爆香蒜末，然后加入蔬菜炒软后放盐；对于炒任何肉，可以提前腌制 15 分钟，并在油中爆香蒜末后下锅；对于煲任何汤，可以加入料酒、姜片、葱段等调味品；对于辣炒类菜肴，则可添加白糖、生抽、姜片等调料；凉拌类则可使用生抽、白糖、醋等配料进行拌匀；韩式拌饭酱需要混合白糖、生抽、辣椒粉等材料制作成调味汁。最后还介绍了其他一些常见的调味汁如雪碧一勺 + 芝麻一勺 + 韩式辣酱一勺以及芝麻一勺 + 芝麻酱一勺 + 盐一勺 + 小米辣两颗等。

https://www.eallion.com/guide-to-cooking/

**Comment by rebron1900** [comment]
以一个偶尔做饭的人来看，这些方法几乎都管用。

**Reply by eallion** [reply]
这样可能没有多好吃，但不会出错。

**Comment by wangyunzi** [comment]
赶紧mark一份，最近天天在家学做菜

**Reply by eallion** [reply]
可以看看 @王刚 @老饭骨 @品诺王 这些真正的厨师的视频，潜移默化中会教很多基础知识。@阿伟 @顾小胖 也不错。
